source/common/tracing/http_tracer_impl.h

```diff
@@ -118,6 +118,7 @@ class EgressConfigImpl : public Config {
     return request_headers_for_tags_;
   }
   bool verbose() const override { return false; }
+  uint32_t maxPathTagLength() const override { return 256; }
```

Comment:
Can we avoid hardcoding 256 in places? Suggest a shared constant.

Reply:
wasn't quite sure where to put it, think I found a reasonable spot. updated useful places to use that constant.